schedule/yast/btrfs/btrfs_sle_libstorage-ng@s390x.yaml

```diff
@@ -0,0 +1,70 @@
+name:           btrfs_libstorage-ng
```

Comment:
could you please add [this ticket](https://progress.opensuse.org/issues/102659) in the PR (and probably assign it and set the same status that for the one here)

Reply:
Added.